docs/user-guide/sdks-using.md

```diff
@@ -40,11 +42,15 @@ If you download the Python SDK packages from Zowe.org, the downloaded folder con
 pip install <fileName>.whl
 ```
 
+### Java
+
+Requires Java runtime version 11 and above.  
+
 ## Getting started
 
 To get started, import the SDK packages to your project. You can pull the packages from an online registry, or download the packages from Zowe.org to install locally.
 
-### Install SDK from online registry
+### Install Node.js and Pyhton SDK from online registry
```

Comment:
*Python

Reply:
done made checkin reflecting the suggestion thanks.